lib/command.c

```diff
@@ -1615,8 +1615,11 @@ permute (struct graph_node *start, struct vty *vty)
       bool skip = false;
       if (stok->type == FORK_TKN && tok->type != FORK_TKN)
         for (ALL_LIST_ELEMENTS_RO (position, ln, gnn))
-           if (gnn == gn && (skip = true))
```

Comment:
Yah since you did the right thing here I assumed you knew; I would prefer to just ignore the warning here since coverity will, as far as I know, only report new bugs. Back this change out and the other one I commented on and we're good to go as far as I'm concerned 🙂